packages/kbn-shared-ux-storybook/src/config/main.ts

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import { defaultConfig } from '@kbn/storybook';
+
+module.exports = {
+  ...defaultConfig,
+  stories: [
+    '../../../kbn-shared-ux*/**/*.stories.+(tsx|mdx)',
+    '../../../../src/plugins/shared_ux/**/*.stories.+(tsx|mdx)',
```

Comment:
It is ok for a package to have a reference to plugin?

Reply:
I can add this to the documentation for this package, to clear up any confusion.